scanners/zap/zap.py

```diff
@@ -393,18 +413,11 @@ def _setup_import_urls(self):
             "parameters": {"fileName": f"{self.container_work_dir}/importUrls.txt"},
         }
 
-        types = ("har", "modsec2", "url", "zap_messages")
-
         source = ""  # Location of the import file on the host
 
         source = self.my_conf("importUrlsFromFile.fileName")
-        if not source:
```

Comment:
This check is already handled in the dataclasses, so it's removed here to avoid duplication